src/Symfony/Component/Translation/Dumper/IcuResFileDumper.php

```diff
@@ -42,6 +42,8 @@ public function formatCatalogue(MessageCatalogue $messages, $domain, array $opti
         $keyTop = $this->getPosition($data);
 
         foreach ($messages->all($domain) as $source => $target) {
+            $target = (string) $target;
```

Comment:
can really the target be null?

Reply:
Strictly speaking, there's no type hinting on MessageCatalogue::$messages
But... Okay, there are supposed to be strings.
Fixed